gitlabAPI.js

```diff
@@ -0,0 +1,32 @@
+const fetch = require("node-fetch");
+
+exports.getProjectsByGroupID = function getProjectsByGroupID(
+  GroupID,
+  key,
+  searchParam
```

Comment:
Please organize the function in better way. JS function param is unnamed, so the ordering is important. We group similar things together, so `key` should not be in the middle of `groupID` and `searchParams`.